dora/core/client/fs/src/main/java/alluxio/client/file/cache/PageMetaStore.java

```diff
@@ -135,6 +137,15 @@ static PageMetaStore create(CacheManagerOptions options) throws IOException {
    */
   void reset();
 
+  /**
+   * Gets all pages by the specified File ID.
+   * @param fileId the target file id
+   * @return a set of PageInfo's of this file, otherwise an empty set if not found
+   */
+  default Set<PageInfo> getAllPagesByFileId(String fileId) {
+    return Collections.emptySet();
+  }
```

Comment:
I think we can make this a required method.

```suggestion
  Set<PageInfo> getAllPagesByFileId(String fileId);
```

Reply:
OK